assignment-tool/tool/src/webapp/vm/assignment/chef_assignments_instructor_list_submissions.vm

```diff
@@ -432,6 +432,9 @@
                  	</a>
 				</p>
 				<div id="allowResubmission_shownInner" style="display:none">
+				#if ($allowReviewService && $assignment.getContent().AllowReviewService)
```

Comment:
There are a lot of checkings like that on this version, as there's no specific way of checking which implementation is being used (besides the title). For future versions it can be corrected.